Extensions/shuffle+.js

```diff
@@ -368,12 +368,12 @@
 	}
 
 	async function fetchArtistTracks(uri) {
-		const { queryArtistDiscographyAll } = Spicetify.GraphQL.Definitions;
+		const { queryArtistOverview } = Spicetify.GraphQL.Definitions;
```

Comment:
`queryArtistOverview` is not available in some of supported spotify versions. revert.